src/defu.ts

```diff
@@ -89,4 +94,16 @@ export const defuArrayFn = createDefu((object, key, currentValue) => {
   }
 });
 
+// Custom version for skipping keys not present in defaults
+export const defuSchema = createDefu((schema, key) => {
+  // Checking for hasOwnProperty is required for compatibility with older browsers
+  if (
+    ('hasOwn' in Object && !Object.hasOwn(schema, key)) ||
```

Comment:
This condition (same as before btw) when there is no runtime support for `Object.hasOwn` completely and silently breaks the purpose of `defuSchema` utility leading to assign all properties for example in (EOL) Node 16. We need to find another way OR simply say this new utility requires `Object.hasOwn` and remove runtime checks.

Reply:
@pi0 Given we're always going to call the global prototype, do you think it's a bad idea to just leave ``hasOwnProperty``? The only reason to use ``hasOwn`` over ``hasOwnProperty`` seems to be when the prototype of an object has been set to null, but our use case is intuitive enough to not be problematic?

Source: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Object/hasOwn#description
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Object#null-prototype_objects

Reply:
I would simply use `in` keyword for property check. `key in schema`. Do you see any downsides with it?

Reply:
@pi0 I always heard it was less strict, since it returns true for prototypes or super classes (i.e 'toString' for instance will always return true for any object)

Reply:
Have you seen https://github.com/unjs/defu/pull/111? We now only merge pure objects only.

Reply:
@pi0 Done!